src/Microsoft.DotNet.GenAPI/Shared/Filters/FilterOutImplicitSymbols.cs

```diff
@@ -0,0 +1,44 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.CodeAnalysis;
+
+namespace Microsoft.DotNet.GenAPI.Shared;
+
+public class FilterOutImplicitSymbols : IAssemblySymbolFilter
+{
+    /// <inheritdoc />
+    public bool Include(INamespaceSymbol ns)
+    {
+        return !ns.IsImplicitlyDeclared;
+    }
```

Comment:
You can also do this:

```suggestion
    public bool Include(INamespaceSymbol ns) => !ns.IsImplicitlyDeclared;
```